BetterMD/__init__.py

```
@@ -1,4 +1,8 @@
 from .elements import *
-from .html import CustomHTML
-from .markdown import CustomMarkdown
-from .rst import CustomRst
\ No newline at end of file
+from .parse import Collection, HTMLParser, MDParser, RSTParser
+
+def from_html(html:'str'):
+    return Symbol.from_html(html)
```

Ruff (0.8.2)

`Symbol` may be undefined, or defined from star imports

(F405)